package.json

```diff
@@ -74,7 +75,6 @@
     "ganache-cli": "6.5.0",
     "merkle-patricia-tree": "2.3.2",
     "openzeppelin-solidity": "2.2.0",
-    "solidity-rlp": "^2.0.0",
-    "truffle-hdwallet-provider": "^1.0.17"
```

Comment:
why remove @?
https://stackoverflow.com/questions/36667258/what-is-the-meaning-of-the-at-prefix-on-npm-packages?noredirect=1&lq=1

Reply:
What? I added `@truffle/hdwallet-provider` and removed `truffle-hdwallet-provider` which is deprecated.